content/en/real_user_monitoring/_index.md

```diff
@@ -80,11 +80,18 @@ Get automated alerts on outliers and groups of errors, timeouts, and crashes to
 
 {{< img src="real_user_monitoring/error_tracking/errors_rum.mp4" alt="RUM error tracking" video=true >}}
 
+## Explore Datadog Session Replay
+
+### Session replays
```

Comment:
```suggestion
### Session replays and developer tools
```

Reply:
we don't need to call it out here, it's not a separate product. i would recommend leaving as is